shoppingcart/models.py

```diff
@@ -59,17 +74,23 @@ def get_registration_code_redemption(cls, code, course_id):
         Returns the registration code redemption object if found else returns None.
         """
         try:
-            code_redemption = cls.objects.get(registration_code__code=code, registration_code__course_id=course_id)
+            code_redemption = cls.objects.get(
+                registration_code__code=code, registration_code__course_id=course_id
+            )
         except cls.DoesNotExist:
             code_redemption = None
         return code_redemption
 
     @classmethod
-    def create_invoice_generated_registration_redemption(cls, course_reg_code, user):  # pylint: disable=invalid-name
+    def create_invoice_generated_registration_redemption(
+        cls, course_reg_code, user
+    ):  # pylint: disable=invalid-name
```

Comment:
I was very curious about the 'invalid-name' here. Turns out there is a 40 char limit on function names. I didn't know that was a Pylint rule.